### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by phoracek**
Fix them to Kubernetes 0.19.0.

Signed-off-by: Petr Horacek <phoracek@redhat.com>

### CODE REVIEW

**Comment by ovirt-infra** [comment]
All tests passed

File: Makefile [hunk 1]
```diff
@@ -4,7 +4,6 @@ IMAGE_TAG ?= latest
 BIN_DIR = $(CURDIR)/build/_output/bin/
 export GOPROXY=direct
 export GOSUMDB=off
-export GOFLAGS=-mod=vendor
```
Comment (RamLavi) [inline Makefile:7]: why is this removed?

File: cluster/kubevirtci.sh [hunk 2]
```diff
@@ -14,7 +14,7 @@
 
 export KUBEVIRT_PROVIDER=${KUBEVIRT_PROVIDER:-'k8s-1.18'}
 
-KUBEVIRTCI_VERSION='df2d0854cffe248800f4eec6b86170fd1187440f'
+KUBEVIRTCI_VERSION='426443f01f541258d17d8abd005c1f9296f614d6'
```
Comment (RamLavi) [inline cluster/kubevirtci.sh:17]: can you please split the PR where the generated files are in a separate commit?
